docs/webformsasinput.md

```diff
@@ -0,0 +1,156 @@
+- They Provide Config/setting options for each Field.
+- They are always associated to Content Types/Bundles. Means all Nodes of the same Content type will share the same modes.
+
+The main difference, other than their purpose (Output v/s Input) is that, on View Modes, the settings you apply to each field are associated to "Formatters" and on Form Modes, the settings you apply to each field are connected to "Widgets".
+
+So, resuming, what lives under the Concept of a "Display Mode": 
+
+## View Mode
+![See all your View Modes](../imgs/view-modes.jpg)
+
+- Each field attached to a Content Entity can have a Formatter applied and most of them have configuration options.
+- Formatters do one thing right: they take the raw, stored value and make it "visible" inside Drupal.
+- Which formatters are available will depend on the "type" of field the Content Entity has. 
+  - E.g A Node title/Label will have a Title formatter with the option of just displaying a text or a text with a link to the entity. 
+  - More Complex and fun Fields, like the ones of type `SBF` will provide a large list of possible `Formatters`, like IIIF driven viewers, Video formatters, Metadata Display (Twig template driven) ones, etc. This is because a SBF type of field has much more than just a text value, it contains a full graph of metadata and properties, inclusive links to Files and provenance metadata.
+
+## Form Mode
+![Form Modes](../imgs/form-modes.jpg)
+
+- Each field attached to a Content Entity can have a Widget applied and most of them have configuration options.
+- Widgets do one thing right: they expose some type of Form/UI interaction that allows a user to input data into the Entity, under that specific field. And of course they make sure that what you input is validated and saved (if good) correctly.
+- Which Widgets are available will depend on the "type" of field the Content Entity has. 
+  - Example: A `Node` Title will have a single Text Input with some options, like the size of the Textfield used to feed it.
+  - More Complex and double fun fields, like the ones of type `SBF`, will provide a larger list of possible Widgets, randing from raw JSON input (like what you would do if your data was already in the right format) to the reason we are reading this: `Webform driven Widgets` like the ones the _webform_strawberrfield_ Drupal module provides, ones that use an existing Webform (which are also Entitites!) you created or we provided as a setting and by that as input to build, massage and enrich, in a visual way, the JSON that SBF type of fields need. Again. This is because a SBF type of field has much more than just a text value, it contains a full graph of metadata and properties, inclusive links to Files and provenance metadata, which for example allows us to use an Upload field directly in the attached/configured webform.
```

Comment:
```suggestion
  - More Complex and fun fields, like the ones of type `SBF` (strawberryfield), will provide a larger list of possible Widgets, ranging from raw JSON input (which you could select if your data was already in the right format) to the reason we are reading this: `Webform driven Widgets`. These Widgets include:
   - ones the _webform_strawberrfield_ Drupal module provides
   - ones that use an existing Webform (which are also Entitites!) which either 1) you created or 2) we provided as a setting
   
If you chose a widget other than the raw JSON, the widget will take the raw JSON to build, massage and enrich the data so that it can be presented in a visual format by the SBF. This is because a SBF type of field has much more than just a text value. It contains a full graph of metadata and properties, inclusive links to Files and provenance metadata, which for example allows us to use an Upload field directly in the attached/configured webform.
```